src/components/menus/IconMenu.tsx

```diff
@@ -23,21 +29,21 @@ const IconMenu = (props: IconMenuProps) => {
         setAnchorEl(null);
     };
 
-    const id = `${props.identifier}-button`;
```

Comment:
This is interesting that this wasn't caught by the linters... I guess it is somewhat edge case?